Krivorotova Tatyana/splitAndMerge.js

```diff
@@ -0,0 +1,11 @@
+/*   splitAndMerge   */
+function splitAndMerge(str, sp){
+    let wrds = str.split(' ');
+    const chars = [];
```

Comment:
maybe something like  `wordsWIthSP` or `modifiedWords`? - `chars` is confusing name